notebooks/1_BayesianWorkflow/3_Ins_EndtoEndWorkflow.ipynb

```diff
@@ -0,0 +1,178 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Section 1.3 End to End Statistics workflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From Wikipedia  \n",
+    "_Statistics is a branch of mathematics dealing with data collection, organization, analysis, interpretation and presentation._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bayesian Versus ...\n",
+    "It should be noted that Bayesian Statistics is one methodology in the broad field of statistics/machine learning. Below we briefly summarize the differences between Bayesian Statistics and other commonly known Statistics/Machine Learning methodologies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Bayesian statistics versus frequentists\n",
+    "**Bayesians:** Given the data how how likely are my parameters?  \n",
+    "**Frequentists:** Given the parameters what is the probability of the observed data?\n",
+    "\n",
+    "#### Frequentist interpretation: How much of a planet is covered in water?\n",
+    "*TODO*: Figure out a way to twist the last problem to meet a frequentist framework"
```

Comment:
Not sure if I understood this, let me know if you need help with this.

Reply:
I need help with this. Basically I'd like to take the globe tossing model and somehow turn it into a frequentist hypothesis test like

```
h0: Planet is not covered with 90% water
h1: Planet is covered with over 90% water

After 10 random samples we conclude with 95% significance that planet has over 90% of water.
```
Something like this so the bayesian approach and frequentist approach can be contrasted

Reply:
I usually do something like this I compare the MLE with the mode of the posterior (generally with 3 priors). And I compare the point estimate with the posterior distribution. Sometimes I also discuss how to get "frequentist" confidence interval but given the time you have I will just compare the MLE with the posterior.